### PR TITLE
close out ffmpeg-4.1 and zstd-1.4.2, both have issued all their PRs

### PR DESCRIPTION
**Pull request by CJ-Wright**

### CODE REVIEW

**Comment by CJ-Wright** [comment]
@conda-forge/core

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-{% set version = "2019.10.11" %}
+{% set version = "2019.10.22" %}
```
Comment (jakirkham) [inline recipe/meta.yaml]: Perhaps we can hold off on the version number bump in this PR. Was planning on bumping the version in PR ( https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/309 ).

Comment (CJ-Wright) [reply recipe/meta.yaml]: ok

Comment (CJ-Wright) [inline recipe/meta.yaml]:
```suggestion
{% set version = "2019.10.11" %}
```

**Comment by jakirkham** [comment]
Thanks @CJ-Wright! 😄

**Pull request merged**